### PR TITLE
Private monica

### PR DESCRIPTION
**Pull request by Monicalyy**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Authorization/Management.Authorization/Generated/AuthorizationManagementClient.cs [hunk 1]
```diff
@@ -8,27 +8,30 @@
...
+	using System.Net.Http;
+	using Microsoft.Azure;
+    using Microsoft.Azure.Management;
     using Microsoft.Rest;
     using Microsoft.Rest.Serialization;
-    using Newtonsoft.Json;
-    using Microsoft.Rest.Azure;
     using Models;
+    using Newtonsoft.Json;
+	using Microsoft.Rest.Azure;
```
Comment (andycnzh) [inline src/SDKs/Authorization/Management.Authorization/Generated/AuthorizationManagementClient.cs]: Keep using in alphabetical order.

File: src/SDKs/Authorization/Management.Authorization/Generated/ProviderOperationsMetadataOperations.cs [hunk 2]
```diff
@@ -282,180 +320,180 @@ internal ProviderOperationsMetadataOperations(AuthorizationManagementClient clie
                 _url += "?" + string.Join("&", _queryParameters);
             }
             // Create HTTP transport objects
-            HttpRequestMessage _httpRequest = new HttpRequestMessage();
+            var _httpRequest = new HttpRequestMessage();
```
Comment (andycnzh) [inline src/SDKs/Authorization/Management.Authorization/Generated/ProviderOperationsMetadataOperations.cs]: Use HttpRequestMessage

File: src/SDKs/Authorization/Management.Authorization/Generated/ProviderOperationsMetadataOperations.cs [hunk 3]
```diff
@@ -527,17 +565,17 @@ internal ProviderOperationsMetadataOperations(AuthorizationManagementClient clie
...
+				_result.RequestId = _httpResponse.Headers.GetValues("x-ms-request-id").FirstOrDefault();
+			}
+			// Deserialize Response
+			if ((int)_statusCode == 200)
             {
                 _responseContent = await _httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
                 try
                 {
-                    _result.Body = SafeJsonConvert.DeserializeObject<Page<ProviderOperationsMetadata>>(_responseContent, this.Client.DeserializationSettings);
+                    _result.Body = Rest.Serialization.SafeJsonConvert.DeserializeObject<Page<ProviderOperationsMetadata>>(_responseContent, Client.DeserializationSettings);
```
Comment (andycnzh) [inline src/SDKs/Authorization/Management.Authorization/Generated/ProviderOperationsMetadataOperations.cs]: Remove Rest.Serialization

**Comment by shahabhijeet** [comment]
@azuresdkci add to whitelist

**Comment by markcowl** [comment]
@Monicalyy You will need to follow the durections here: https://aka.ms/azuregithub for joining the Azure organization, or, if you are not part of the azure organization, sign the CLS.

@darshanhs90  Please approve

**Comment by andycnzh** [comment]
This is obsolete, so close it.

**Pull request closed without merging**